python/restate/context.py

```diff
@@ -282,13 +293,13 @@ def key(self) -> str:
     @abc.abstractmethod
     def get(self,
             name: str,
-            serde: Serde[T] = JsonSerde()) -> Awaitable[Optional[Any]]:
+            serde: Serde[T] = JsonSerde()) -> RestateDurableFuture[Optional[Any]]:
         """
         Retrieves the value associated with the given name.
         """
 
     @abc.abstractmethod
-    def state_keys(self) -> Awaitable[List[str]]:
+    def state_keys(self) -> RestateDurableFuture[List[str]]:
```

Comment:
in other SDKs this is not combineable, because generally it's not something we expect people to do...